files/www/cgi-bin/setup

```diff
@@ -1087,6 +1093,7 @@ if wifi_enable == "1" then
     do
         html.print("<option value='" .. width .. "'".. (wifi_chanbw == tostring(width) and " selected" or "") .. ">" .. width .. " MHz</option>")
     end
+    html.print("<tr><td>SSID</td><td><input type=text size=15 name=wifi_ssid value='" .. wifi_ssid .. "'>-" .. wifi_chanbw .. "-v3</td></tr>")
```

Comment:
This looks like its in the wrong place? It seems to be inside the <select> ... </select> now.